f/radios/acre2/acre2_clientInit.sqf

```diff
@@ -119,3 +130,103 @@ if(_typeOfUnit != "NIL") then {
 
   };
 };
+
+// ====================================================================================
+
+// ASSIGN DEFAULT CHANNELS TO RADIOS
+// Depending on the squad joined, each radio is assigned a default starting channel
+
+if(!f_radios_settings_acre2_disableRadios) then {
+
+	private ["_presetArray","_presetLRArray","_radioSR","_radioLR","_radioExtra","_hasSR","_hasLR","_hasExtra","_groupID","_groupIDSplit","_groupChannelIndex","_groupLRChannelIndex","_groupName"];
+
+	if (_radiosAdded != 0) then {
+		waitUntil {uiSleep 0.3; count ([] call acre_api_fnc_getCurrentRadioList) >= _radiosAdded};
```

Comment:
Copying @Sniperhid comment from old PR: "might be worth using [] call acre_api_fnc_isInitialized; over acre_api_fnc_getCurrentRadioList as it already waits for all radios to be properly setup. Also negating the need for _radiosAdded"